components/board/beaglebone/board.go

```diff
@@ -22,5 +22,5 @@ func init() {
 		golog.Global().Debugw("error getting beaglebone GPIO board mapping", "error", err)
 	}
 
-	genericlinux.RegisterBoard(modelName, gpioMappings, true)
+	genericlinux.RegisterBoard(modelName, gpioMappings, /* usePeriph = */ false)
```

Comment:
nicely done, so this is verified as a non-periph board now?

Reply:
Yup! Testing every single pin is a real pain, so I did all of them before changing this line, but then only 20ish (including all hardware PWM pins) afterwards.

Reply:
(to clarify: I think we still use Periph for I2C and SPI. but not for GPIO!)